### PR TITLE
Updated German translation

### PR DESCRIPTION
**Pull request by grand-piano**
The most important change is to translate the term **'levels'** into **'Tonwerte'** instead of 'Werte' and **'core options'** into **'Kern-Optionen'** instead of 'Zentrale Optionen'. The other changes are mainly capitalization changes at the beginning of the sentence and the correction of some misspelled lowercase nouns, which are capitalized in german

### CODE REVIEW

File: po/de.po [hunk 1]
```diff
@@ -1357,7 +1357,7 @@ msgstr[1] "Sollen diese leeren Verzeichnisse entfernt werden?"
 #: ../src/common/film.c:314
 msgid "remove empty directory?"
 msgid_plural "remove empty directories?"
-msgstr[0] "leeres Verzeichnis entfernen?"
+msgstr[0] "Leeres Verzeichnis entfernen?"
 msgstr[1] "leere Verzeichnisse entfernen?"
```
Comment (ssbssa) [inline po/de.po]: Why is the plural (here and the later ones) not capitalized as well?

File: src/bauhaus/bauhaus.c [hunk 2]
```diff
@@ -442,10 +442,10 @@ void dt_bauhaus_init()
   gtk_widget_set_name(darktable.bauhaus->popup_area, "bauhaus-popup");
 
   darktable.bauhaus->line_space = 2;
-  darktable.bauhaus->line_height = 11;
-  darktable.bauhaus->marker_size = 0.3f;
-  darktable.bauhaus->label_font_size = 0.6f;
-  darktable.bauhaus->value_font_size = 0.6f;
+  darktable.bauhaus->line_height = 13;
```
Comment (ssbssa) [inline src/bauhaus/bauhaus.c]: What do the changes in bauhaus.c have to do with the german translation?

Comment (grand-piano) [reply src/bauhaus/bauhaus.c]: Something's gone wrong. Will delete the pull request and create a new one.

File: po/de.po (hunk 1 above)
Comment (grand-piano) [reply po/de.po]: I didn't see that. I will fix it

**Pull request closed without merging**